docs/detections/rules-ui-manage.asciidoc

```diff
@@ -173,25 +176,40 @@ TIP: You can also use {kib}'s {kibana-ref}/managing-saved-objects.html#managing-
+. Do one of the following:
+** Export a single rule: Find the rule in the Rules table, then select  **All actions** ->**Export**. Alternatively, export the rule from its details page (click on the rule name to open its details, then click **All actions** ->**Export**).
+** Export multiple rules: In the Rules table, select the rules you want to export, then click **Bulk actions -> Export**.
 
-.. Click *Import rules*.
-.. Drag and drop the file that contains the detection rules.
-+
-NOTE: Imported rules must be in an `.ndjson` file.
-.. (Optional) Select *Overwrite existing detection rules with conflicting "rule_id"* to update existing rules if they match the `rule_id` value of any rules in the import file. Configuration data included with the rules, such as actions, is also overwritten.
-.. (Optional) Select *Overwrite existing exception lists with conflicting "list_id"* to replace existing exception lists with exception lists from the import file if they have a matching `list_id` value.
-.. (Optional) Select *Overwrite existing connectors with conflicting action "id"* to update existing connectors if they match the `action id` value of any rule actions in the import file. Configuration data included with the actions is also overwritten.
+The rules are exported to an `.ndjson` file.
+
+[float]
+[[import-rules-ui]]
+==== Import rules
+
+. Above the Rules table, click *Import rules*.
+. In the Import rules modal: 
+.. Drag and drop the `.ndjson` file that contains the exported rules. 
+.. (Optional) Select the appropriate options to overwrite existing data:
+** *Overwrite existing detection rules with conflicting "rule_id"*: Updates existing rules if they match the `rule_id` value of any rules in the import file. Configuration data included with the rules, such as actions, is also overwritten.
+** *Overwrite existing exception lists with conflicting "list_id"*: Replaces existing exception lists with exception lists from the import file if they have a matching `list_id` value.
+** *Overwrite existing connectors with conflicting action "id"*: Updates existing connectors if they match the `action id` value of any rule actions in the import file. Configuration data included with the actions is also overwritten.
 .. Click *Import rule*.
-.. (Optional) If a connector is missing sensitive information after the import, a warning displays and you're prompted to fix the connector. In the warning, click *Go to connector*. On the Connectors page, find the connector that needs to be updated, click *Fix*, then add the necessary details.  
+
+The imported rules are added to the Rules table. 
+
+[NOTE] 
+=====
+If the prebuilt rule package doesn't have the original version of a rule that you're importing, the rule is marked as `Modified` when both the following criteria are met:
+
+- The rule's ID (`rule_id`) is identical to that of an already installed prebuilt rule.
+- The imported rule's settings are different from the currently installed rule. 
+=====
```

Comment:
There are many more edge cases besides this one that we have to handle. We're currently trying to cover them all in this [PR](https://github.com/elastic/kibana/pull/213434) in a test plan under the "Handling missing base versions" and other sections. I don't think we should attempt to describe any of this under-the-hood complexity in the user-facing docs. The user shouldn't have to deal with this complexity and shouldn't even know that there's a package with prebuilt rules. The package is the implementation details of the prebuilt rules delivery mechanism.

I'd suggest to delete this note from the docs.

Reply:
Sounds good - I debated about adding this note, but am fine to remove it. 

Removed in [bb2e2cc](https://github.com/elastic/security-docs/pull/6563/commits/bb2e2cc0cb8a081dad416e0f58e86bb885234439).